freetype-config was removed from oe-core in

commit 5870bd272b0b077d0826fb900b251884c1c05061
Author: Richard Purdie <richard.purdie@linuxfoundation.org>
Date:   Thu May 22 10:59:33 2014 +0100

    binconfig-disabled: Add class and use

diff --git a/configure.in b/configure.in
index 408e8d6..5f9e730 100644
--- a/configure.in
+++ b/configure.in
@@ -86,39 +86,11 @@ case "$host" in
 esac
 AM_CONDITIONAL(USE_VERSION_RC, test x$use_version_rc = xtrue)
 
-dnl Check for the FreeType 2 library
-dnl
-dnl Get the cflags and libraries from the freetype-config script
-dnl
-AC_ARG_WITH(freetype-prefix,[  --with-freetype-prefix=PFX   Prefix where FREETYPE is 
-installed (optional)],
-            freetype_prefix="$withval", freetype_prefix="")
-AC_ARG_WITH(freetype-exec-prefix,[  --with-freetype-exec-prefix=PFX Exec prefix 
-where FREETYPE is installed (optional)],
-            freetype_exec_prefix="$withval", freetype_exec_prefix="")
-
-if test x$freetype_exec_prefix != x ; then
-     freetype_args="$freetype_args --exec-prefix=$freetype_exec_prefix"
-     if test x${FREETYPE_CONFIG+set} != xset ; then
-        FREETYPE_CONFIG=$freetype_exec_prefix/bin/freetype-config
-     fi
-fi
-if test x$freetype_prefix != x ; then
-     freetype_args="$freetype_args --prefix=$freetype_prefix"
-     if test x${FREETYPE_CONFIG+set} != xset ; then
-        FREETYPE_CONFIG=$freetype_prefix/bin/freetype-config
-     fi
-fi
-AC_PATH_PROG(FREETYPE_CONFIG, freetype-config, no)
-no_freetype=""
-if test "$FREETYPE_CONFIG" = "no" ; then
-    AC_MSG_ERROR([
-*** Unable to find FreeType2 library (http://www.freetype.org/)
-])
-else
-    CFLAGS="$CFLAGS `$FREETYPE_CONFIG $freetypeconf_args --cflags`"
-    LIBS="$LIBS `$FREETYPE_CONFIG $freetypeconf_args --libs`"
-fi
+PKG_CHECK_MODULES(FREETYPE2, freetype2, 
+    CFLAGS="$CFLAGS $FREETYPE2_CFLAGS"
+    LIBS="$LIBS $FREETYPE2_LIBS",
+    AC_MSG_ERROR([*** Unable to find FreeType2 library (http://www.freetype.org/)])
+)
 
 dnl Check for SDL
 SDL_VERSION=2.0.0
